host/src/lib/HelperFunctions.jsx

```diff
@@ -329,4 +365,26 @@ export const buildVictoryDataObjectShortAnswer = (
         answerCorrect: answer.isCorrect
     }))
   ]
-};
\ No newline at end of file
+};
+
```

Comment:
similarly, we need to build data objects for Victory that reflect the short answer phase two data